### PR TITLE
rename package 3dt -> dcos-diagnostics

### PR DESCRIPTION
**Pull request by darkonie**
## High Level Description

[ee PR](https://github.com/mesosphere/dcos-enterprise/pull/1070)

rename package `3dt` to `dcos-diagnostics`

## Related Issues

  - [DCOS-13066](https://jira.mesosphere.com/browse/DCOS-13066)

## Checklist for all PR's

  - [X] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: the current integration tests are covering all the functionality.
  - [X] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [X] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [X] Change log from the last version integrated (this should be a link to commits for easy verification and review): [dcos-diagnostics](https://github.com/dcos/dcos-diagnostics/compare/6b560880d5a2419b8bad387d06d9355d3ad207ab...1e30ef651ec47910f9207192e0276df3bd239b8a)
  - [X] Test Results:  [jenkins](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-diagnostics-pulls/3/console)
  - [X] Code Coverage (if available): [jenkins](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-diagnostics-pulls/3/console)
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and Review Process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using github's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by darkonie** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by darkonie** [comment]
@mesosphere-mergebot label Ship It

**Comment by amitaekbote** [comment]
@mesosphere-mergebot label ship it

**Comment by amitaekbote** [comment]
We will need to update https://github.com/dcos/dcos/blob/master/dcos_installer/upgrade.py#L75 once this pr lands

**Comment by darkonie** [comment]
squashed commits and updated dcos-diagnostics to the latest sha to include https://github.com/dcos/dcos-diagnostics/pull/2

File: packages/dcos-diagnostics/buildinfo.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
   "single_source" : {
     "kind": "git",
-    "git": "https://github.com/dcos/3dt.git",
-    "ref": "6b560880d5a2419b8bad387d06d9355d3ad207ab",
+    "git": "https://github.com/dcos/dcos-diagnostics.git",
+    "ref": "ec7fb9d99fcb77120978f7dfbe382aca7702102d",
     "ref_origin": "master"
   },
-  "username": "dcos_3dt",
+  "username": "dcos_diagnostics",
```
Comment (adam-mesos) [inline packages/dcos-diagnostics/buildinfo.json]: Is there a risk that files previously owned by the `dcos_3dt` user will no longer be accessible?

File: packages/dcos-diagnostics/buildinfo.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
   "requires": ["exhibitor"],
   "single_source" : {
     "kind": "git",
-    "git": "https://github.com/dcos/3dt.git",
-    "ref": "6b560880d5a2419b8bad387d06d9355d3ad207ab",
+    "git": "https://github.com/dcos/dcos-diagnostics.git",
+    "ref": "ec7fb9d99fcb77120978f7dfbe382aca7702102d",
```
Comment (adam-mesos) [inline packages/dcos-diagnostics/buildinfo.json]: FYI: This will conflict with https://github.com/dcos/dcos/pull/1671

Comment (darkonie) [inline packages/dcos-diagnostics/buildinfo.json]: if we ship this PR as part of the next train we need to close [#1671](https://github.com/dcos/dcos/pull/1671)

**Comment by amitaekbote** [comment]
Marked #1671 as ready for review, once this merges we can decide its fate

File: packages/dcos-diagnostics/buildinfo.json (hunk 1 above)
Comment (darkonie) [inline packages/dcos-diagnostics/buildinfo.json]: the bootstrap will create new directories and generate new service accounts, so we should be all good. I do not see any files that 3dt might own. Maybe diagnostics bundles.. The only reason why customer would care about bundles is when we asked them to provide it for us. So it should not be a problem.

**Comment by darkonie** [comment]
3dt does not have any persistent data. When the cluster is upgraded all new service accounts will be provisioned for `dcos-diagnostics` and new linux user added. After the restart renamed `dcos-diagnostics` service should pick up all configs from new location and work as expected.

cc @adam-mesos

**Pull request merged**